department-of-veterans-affairs/caseflow
# Display Attorney Notes for Judge Decision Review for Legacy Appeals

Resolves [CASEFLOW-2736](https://vajira.max.gov/browse/CASEFLOW-2736)

### Description
Added method to the `LegacyTaskSerializer`  that appends the saved attorney notes when serializing the task data to be sent to the frontend. On the frontend added a key displaying the notes.

### Acceptance Criteria
- [ ] Notes left by the attorney during decision drafting step for legacy appeals are displayed to the judge on their decision review task.

### Testing Plan

Make sure your reset you local DB `make reset` as this PR has some changes to the seed file.

1. Switch views
2. Select “Assign BVAAABSHIRE team cases”
3. Select a legacy case (the first one in list after make reset) and note the case ID
4. Check box and assign it to user “Ezra M Erdman”
5. Switch users to BVAEERDMAN , go to queue
6. Select the legacy appeal that was just assigned
7. For “Draft decision” task, select “Decision ready for review”
8. Select “No Special Issues” and click “Continue”
9. Select dispositions for all issues and click “Continue”
10. Add a document ID (format: 12345-12345678) and fill out something in the notes field
11. Click “Continue” to finish the process
12. Switch back to BVAAABSHIRE and go to Queue
13. Find the case that you have been using and open it to Case Details
14. Verify that you see a “View task instructions” link for the “Review” task
15. Click link and ensure you see the notes from the attorney review

### User Facing Changes

 BEFORE|AFTER
 ---|---
![Screen Shot 2022-01-13 at 10 27 55 AM](https://user-images.githubusercontent.com/87207304/149359367-875cac20-f9fb-46a7-8101-1ab50afc9b16.png) | ![Screen Shot 2022-01-13 at 10 29 09 AM](https://user-images.githubusercontent.com/87207304/149359449-4aff6a0c-6518-4701-82b5-039f7fd48666.png)
